demo/protobuf_getting_started/update.sh

```diff
@@ -1,21 +1,20 @@
-#!/bin/bash
-
-clean() {
```

Comment:
What's the logic here not to clean up the deps and public? It generally seems useful to do so.

Reply:
I deleted it because I don't see any `public` or `deps` folder inside of `demo/protobuf_getting_started`.